language/ko.json

```diff
@@ -0,0 +1,341 @@
+{
+  "semantics": [
+    {
+      "label": "미디어",
+      "fields": [
+        {
+          "label": "유형",
+          "description": "문제 위에 표시할 선택적 미디어"
+        },
+        {
+          "label": "이미지 확대/축소 사용 안 함"
+        }
+      ]
+    },
+    {
+      "label": "태스크 설명",
+      "description": "과제를 여기에 설명합니다.",
+      "placeholder": "잘 듣고 받아 쓰세요."
+    },
+    {
+      "label": "문장",
+      "widgets": [
+        {
+          "label": "기본설정"
+        }
+      ],
+      "entity": "문장",
+      "field": {
+        "label": "문장",
+        "fields": [
+          {
+            "label": "설명",
+            "description": "선택사항으로 텍스트 입력 필드 위에 간단한 설명을 넣을 수 있습니다.(예> 대화에 유용합니다.)"
+          },
+          {
+            "label": "소리 샘플",
+            "description": "정상 속도로 말하는 문장"
+          },
+          {
+            "label": "느린 소리 샘플,
```

Comment:
@totoromaum This is not valid json, because you are missing a `"` at the end of this line. Please note that github hints to this problem by marking the following sentences in red. If you want to keep on translating here on github which has it's pitfalls like this one, please double check your pull requests. You can, for instance, use https://jsonchecker.com/ and paste a translation file there. It will tell you if it's valid json format, thus if at least the syntax is correct. As I mentioned before, using https://translate-h5p.tk/ may be a good option.

Reply:
@otacke, thank you for letting me know. I will double check before I make the pull request. Have a great one!